language/testing-infra/diem-transactional-test-runner/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "diem-transactional-test-runner"
```

Comment:
do we need to put this in a separate dir for move/diem separation? Like, should it be in the language/ dir?

And minor nit, but maybe `diem-transactional-test-harness` not sure it really matters though

Reply:
Eventually we'll definitely move this away current location, but I don't think now is the right time. I think we should at least come up with some rough guidelines on how we relocate diem crates in language before we start to tackle individual cases.

I'll change the name to `diem-transactional-test-harness` as it sounds slightly more accurate.